woocommerce/woocommerce
# Update Payfast logo and title

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Closes #37752.

Payfast has gone through a rebranding that updates their logo and also a name change from “PayFast” to “Payfast”.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to `WooCommmerce > Settings` and Change store country to `South Africa`
2. Go to `WooCommerce > Settings > Advanced > WooCommerce.com` (/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com)
3. Uncheck `Display suggestions within WooCommerce` option
4. Go to `WooCommerce > Home` and click on `Set up payments` task from Task list
5. Observe that the payfast name and logo are shown as below 

![Screenshot 2023-05-03 at 16 38 17](https://user-images.githubusercontent.com/4344253/235869067-1976f9ac-601e-4756-a616-9ec89b11cc0b.png)
